CMakeLists.txt

```diff
@@ -25,11 +25,18 @@ set(ALICEVISION_BUNDLE_PREFIX "${CMAKE_INSTALL_PREFIX}/bundle" CACHE STRING "Pat
 
 set(ALICEVISION_ROOT ${PROJECT_BINARY_DIR})
 
+# ==============================================================================
+# GNUInstallDirs CMake module
+# - Define GNU standard installation directories
+# - Provides install directory variables as defined by the GNU Coding Standards.
+# ==============================================================================
+include(GNUInstallDirs)
+
```

Comment:
Would be nice to add an option to disable it:
`option(ALICEVISION_USE_RPATH "Add RPATH on softwares with relative paths to libraries" ON)`

```
if(ALICEVISION_USE_RPATH)
...
endif()
```

Reply:
`software`, it is uncountable